driver.h

```diff
@@ -0,0 +1,121 @@
+/* SPDX-License-Identifier: (GPL-2.0 OR BSD-3-Clause)
+ *
+ *  This file is provided under a dual BSD/GPLv2 license.  When using or
+ *  redistributing this file, you may do so under either license.
+ *
+ *  Copyright(c) 2019 Intel Corporation. All rights reserved.
+ *
+ *  Author: Marcin Zielinski <marcinx.zielinski@linux.intel.com>
+ */
+
+#ifndef DRIVER_H
+#define DRIVER_H
+
+#include "ioctl.h"
+
+#include <linux/cdev.h>
+#include <linux/pci.h>
+#include <linux/uaccess.h>
+
+#ifndef PCIBIOS_SUCCESSFUL
```

Comment:
@lgirdwood Unfortunately, publishing the Python test scripts doesn't lie within my competence, so I think this could be discussed separately.

_PCIBIOS_SUCCESSFUL_ is a kernel's constant indicating a successful PCI operation; it's defined in _linux/pci.h_. I use it in the code in PCI-related operations instead of checking/returning just 0 to make the code clearer. I really doubt it will ever be changed; the check in the header is just in case :)

Reply:
Please use static_assert or remote it at all.

Reply:
Done.